AlphaWalletTests/Tokens/Coordinators/GetIsERC1155ContractCoordinatorTestCase.swift

```diff
@@ -30,13 +30,18 @@ class GetIsERC1155ContractCoordinatorTestCase: XCTestCase {
     func testgetIsERC1155Contract() throws {
         let coordinator = GetIsERC1155ContractCoordinator(forServer: .main, cacheName: fileName)
         let expectation = expectation(description: "Waiting for server response")
-        coordinator.getIsERC1155Contract(for: address1)
-            .done { returnValue in
-                self.result = returnValue
-                expectation.fulfill()
-            }
-            .cauterize()
-        wait(for: [expectation], timeout: 10 * 60)
+        var fetchedError: Error?
+        firstly {
+            coordinator.getIsERC1155Contract(for: address1)
+        }.done { returnValue in
+            self.result = returnValue
+            expectation.fulfill()
```

Comment:
If https://github.com/AlphaWallet/alpha-wallet-ios/pull/4422#discussion_r858264456 works, would you apply it here too?

Reply:
Applied.